bundle/manifests/nvidia-network-operator.clusterserviceversion.yaml

```diff
@@ -608,15 +584,4 @@ spec:
   provider:
     name: NVIDIA
     url: https://github.com/Mellanox/network-operator/
-  relatedImages:
```

Comment:
related images shouldn't be deleted

Reply:
Seems that the operator-sdk is now ignoring the relatedImages from the `config/manifests/bases`
It used to work in previous version...

Reply:
it stopped working in operator-sdk v1.19.1